en/docs/concepts/rate-limiting.md

```diff
@@ -1,10 +1,17 @@
 # Rate Limiting
 
-The WSO2 API Microgateway by default has an inbuilt mechanism to handle rate limiting. When a Microgateway project is initialized, an execution plan related to the subscription-level throttling policy defined in the API is created in-memory. The API Microgateway monitors the incoming requests and publishes the events to a stream defined in the inbuilt stream processing engine. This stream is then read and fed into the relevant pre-defined execution plan. The execution plan detects the throttling point and publishes an event to a stream (globalThrottleStream). When the globalThrottleStream is updated, the API Microgateway updates its local counters in the background as the gateway is subscribed to the global throttle stream.
+The WSO2 API Microgateway by default has an inbuilt mechanism to handle rate limiting. When creating an API, users can define subscription-level throttling policies. When a Microgateway project gets initialized, an execution plan related to the subscription-level throttling policy gets created in memory. The API Microgateway monitors the incoming requests and publishes the events to a stream defined in the inbuilt stream processing engine. This stream is then read and fed into the relevant pre-defined execution plan. The execution plan detects the throttling point and publishes an event to a stream (globalThrottleStream). When the globalThrottleStream is updated, the API Microgateway updates its local counters in the background as the gateway is subscribed to the global throttle stream.
```

Comment:
This content is invalid. This is not how local throttling works at the moment.
And also talking about local throttling here and again mentioning local and distributing throttling below is confusing. Talk generally about rate limiting here and explain each in the below sub topics

Reply:
`The API Microgateway monitors the incoming requests and publishes the events to a stream defined in the inbuilt stream processing engine. This stream is then read and fed into the relevant pre-defined execution plan. The execution plan detects the throttling point and publishes an event to a stream (globalThrottleStream). When the globalThrottleStream is updated, the API Microgateway updates its local counters in the background as the gateway is subscribed to the global throttle stream.` Is this content also invalid?